extensions/wikia/PhalanxII/tests/PhalanxServiceTest.php

```diff
@@ -13,16 +13,16 @@ class PhalanxServiceTest extends WikiaBaseTest {
 	 * setup tests
 	 */
 	public function setUp() {
-		$this->setupFile =  dirname(__FILE__) . '/../Phalanx_setup.php';
+		$this->setupFile =  dirname( __FILE__ ) . '/../Phalanx_setup.php';
 		parent::setUp();
 		$this->checkPhalanxAlive();
 	}
 
 	public function checkPhalanxAlive( ) {
 		$this->service = new PhalanxService();
-		if (!$this->service->status()) {
-			//Skip test if phalanx service is not available
-			throw new Exception("Can't connect to phalanx service");
+		if ( !$this->service->status() ) {
+			// Skip test if phalanx service is not available
+			throw new Exception( "Can't connect to phalanx service" );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS112)